### PR TITLE
Added tags to project templates to make it easier to find them

### PR DESCRIPTION
**Pull request by frobijn**
## Motivation and Context
The NF projects can now only be found by typing `nanoframework` in the VS2022 UI. That is very clumsy, VS2022 has a much better UI-native way to find projects.

See [Microsoft documentation](https://learn.microsoft.com/en-us/visualstudio/ide/template-tags?view=vs-2022).

Although VS2019 version 16.1+ supports tags as well, no changes are made to the VS2019 templates, as it is not clear to me whether the nanoframework extension should support earlier versions as well.

## Types of changes
- [x] Improvement (non-breaking change that improves a feature, code or algorithm)
- [ ] Bug fix (non-breaking change which fixes an issue with code or algorithm)
- [x] New feature (non-breaking change which adds functionality to code)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Config and build (change in the configuration and build system, has no impact on code or features)
- [ ] Dependencies (update dependencies and changes associated, has no impact on code or features)
- [ ] Unit Tests (add new Unit Test(s) or improved existing one(s), has no impact on code or features)
- [ ] Documentation (changes or updates in the documentation, has no impact on code or features)

## Checklist:
- [ ] My code follows the code style of this project (only if there are changes in source code).
- [ ] My changes require an update to the documentation (there are changes that require the docs website to be updated).
- [ ] I have updated the documentation accordingly (the changes require an update on the docs in this repo).
- [x] I have read the [CONTRIBUTING](https://github.com/nanoframework/.github/blob/main/CONTRIBUTING.md) document.
- [ ] I have tested everything locally and all new and existing tests passed (only if there are changes in source code).
- [ ] I have added new tests to cover my changes.

### CODE REVIEW

**Comment by frobijn** [comment]
@dotnet-policy-service agree

**Review by josesimoes — CHANGES_REQUESTED** [review]
Nice improvement!
These were "just" migrated from VS2019, therefore this tag thing was overlooked.

And yes, we're still supporting VS2019. Can you please add these to the templates?

File: CSharp.BlankApplication/CS.BlankApplication-vs2022.vstemplate [hunk 1]
```diff
@@ -11,6 +11,10 @@
     <DefaultName>NFApp</DefaultName>
     <ProvideDefaultName>true</ProvideDefaultName>
     <PromptForSaveOnCreation>true</PromptForSaveOnCreation>
+	<LanguageTag>CSharp</LanguageTag>
+	<PlatformTag>Nanoframework</PlatformTag>
```
Comment (josesimoes) [inline CSharp.BlankApplication/CS.BlankApplication-vs2022.vstemplate:15]: Mind the case (for coherency) 
```suggestion
	<PlatformTag>nanoFramework</PlatformTag>
```

**Review by josesimoes — APPROVED** [review]
Perfect! Thanks.

**Pull request merged**